groups/templates/groups/emails/new_discussion.txt

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}{% blocktrans with username=user.get_full_name name=discussion.name group=discussion.group.name creator=discussion.creator.name %}Dear {{ username }},
 
-A new discussion "{{ name }}" has been posted to the group {{ group }} by {{ creator }}{% endblocktrans %}{% if discussion.comments.first.body %}:
+A new discussion on "{{ name }}" has been posted to the {{ group }} group by {{ creator }}{% endblocktrans %}{% if discussion.comments.first.body %}:
```

Comment:
I thought about that, but I think groups will have names that fit this pattern well enough that it should work out.  `...been posted to the Haematology group`, that sort of thing.  I'm happy to add the quotes if you reckon they'll be needed though :)